### PR TITLE
Prepare for new stemcell release

### PR DESCRIPTION
**Pull request by liukai**
cc @jtai @alexism 

I'll cut a new release in github after this PR is merged. Also how can I update the version here: https://rubygems.org/gems/stemcell ?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    stemcell (0.8.0)
+    stemcell (0.8.2)
```
Comment (jtai) [inline Gemfile.lock]: Where did 0.8.2 come from?

**Comment by liukai** [comment]
@igor47 can you give me some guidance to update the gem?

**Comment by alexism** [comment]
Create an account at https://rubygems.org/sign_up using your personal email and I'll add you as an owner.

**Comment by liukai** [comment]
Hey folks, here are my action items:

- [x] bump version and run necessary changes on Gemfile and change log.
- [x] all specs pass
- [ ] cut a new git release: I'll do this after merge this PR and use github's "release" feature to cut new tag and add release notes.

Is there any other thing I'd do?

**Comment by jtai** [comment]
Seems like the build is erroring out on ohai 0.8. Perhaps we can just upgrade stemcell itself without bumping all the other dependencies?

**Comment by jtai** [comment]
@liukai I pushed an alternate commit to your branch that should fix the issues. Please double-check it, then merge and follow up with the release steps.

**Pull request merged**